Jenkinsfile

```diff
@@ -126,18 +126,18 @@ pipeline {
                     sh "env; echo Building BUILD_IMAGE=${env.BUILD_IMAGE} for BUILD_TARGET=${env.BUILD_TARGET}, CHANGE_ID=${env.CHANGE_ID}, CACHEBUST=${env.CACHEBUST}."
                     sh "docker build --cache-from ${env.BUILD_IMAGE} --label cachebust=${env.CACHEBUST} --target build --build-arg CACHEBUST=${env.CACHEBUST} -t ${env.BUILD_IMAGE} ."
                     // Create a running container (https://stackoverflow.com/a/38308399).
-                    sh "docker create --name=builder -it ${env.BUILD_IMAGE}"
+                    sh "docker create --name=builder ${env.BUILD_IMAGE} sleep infinity"
                     sh "docker start builder && \
-                        i=0; \
+                        timeout=0; \
                         while [ \$(docker inspect --format '{{.State.Status}}' builder) != running ]; do \
                             docker ps; \
-                            if [ \$i -gt 7 ]; then \
+                            if [ \$timeout -ge 60 ]; then \
                                docker inspect builder; \
                                echo 'ERROR: builder container still not running'; \
                                exit 1; \
                             fi; \
                             sleep 10; \
-                            i=\$((\$i + 1)); \
+                            timeout=\$((\$timeout + 10)); \
```

Comment:
Doesn't such double-parentheses expression work without inner $-sign equally?

Reply:
What do you mean with `inner $-sign`?

```
$ echo ((1 + 1))
-bash: syntax error near unexpected token `('
$ echo $((1 + 1))
2
```

Reply:
I mean, one can use $((timeout + 10)) instead of $(($timeout + 10)) ?
"inner" was referring to $-sign in the parenthesis

Reply:
Apparently one can. Learned something new again...

However, I'd prefer to not start another test run with this change and just get it merged as-is.

Reply:
It failed because of https://github.com/intel/pmem-csi/issues/485, so I might as well update it once more...